app/assets/v2/images/Lightbulb.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This needs squashed.  You can do that via `make compress-images` or simply `svgo app/assets/v2/images/Lightbulb.svg` for the time being.